File: system-test/dotnet/Dsl.Core/Shared/ResponseVerification.cs
namespace Dsl.Core.Shared;

public class ResponseVerification<TResponse>
{
    public ResponseVerification(TResponse response, UseCaseContext context)
    {
        Response = response;
        Context = context;
    }

    protected TResponse Response { get; }
    protected UseCaseContext Context { get; }
}

File: system-test/java/src/main/java/com/mycompany/myshop/testkit/dsl/core/shared/ResponseVerification.java
package com.mycompany.myshop.testkit.dsl.core.shared;

public class ResponseVerification<TResponse> {
    private final TResponse response;
    private final UseCaseContext context;

    public ResponseVerification(TResponse response, UseCaseContext context) {
        this.response = response;
        this.context = context;
    }

    protected TResponse getResponse() {
        return response;
    }

    protected UseCaseContext getContext() {
        return context;
    }
}



